sys/fido2/ctap/ctap.c

```diff
@@ -282,7 +272,12 @@ int fido2_ctap_init(void)
         return -EPROTO;
     }
 
-    ret = _read_state_from_flash(&_state);
+    /**
+     * CTAP state information is stored at flashpage 0 of the memory area
```

Comment:
Is it really flashpage 0 or offset zero?

Reply:
Both. It is stored at offset 0 of flashpage 0